hms-app/src/components/card-details/UserDetails.tsx

```diff
@@ -71,15 +70,15 @@ export default function UserDetails(props: { userID: string }) {
   }
 
   const deleteSelectedUser = () => {
-    deleteUser(userID).then((data) => {
+    deleteUser(userId).then((data) => {
       setDeleteModalOpened(false)
     })
   }
 
   useEffect(() => {
     loadSelectedUser()
     setIsUserLoading(true)
```

Comment:
I know this isn't part of this PR, but this feels wrong.  By the time that `loadSelectedUser()` finishes execution the user is fully loaded.  However, right after that we set the isLoading flag for the user to true, which implies it is still loading.  Instead, the first line in the `loadSelectedUser()` function should set that value to true